pkgs/applications/networking/protonvpn-gui/default.nix

```diff
@@ -1,74 +1,98 @@
 { lib
 , buildPythonApplication
 , fetchFromGitHub
-, wrapGAppsHook
-, gdk-pixbuf
-, glib-networking
 , gobject-introspection
-, imagemagick
-, librsvg
-, pango
-, webkitgtk
-# Python libs
-, protonvpn-nm-lib
-, psutil
-# Optionals
+, wrapGAppsHook
+, dbus-python
+, packaging
+, proton-core
+, proton-keyring-linux
+, proton-keyring-linux-secretservice
+, proton-vpn-api-core
+, proton-vpn-connection
+, proton-vpn-killswitch
+, proton-vpn-killswitch-network-manager
+, proton-vpn-logger
+, proton-vpn-network-manager
+, proton-vpn-network-manager-openvpn
+, proton-vpn-session
+, pycairo
+, pygobject3
+, pytest-cov
+, pytestCheckHook
 , withIndicator ? true
-, libappindicator-gtk3 }:
+, libappindicator-gtk3
+, libayatana-appindicator
+}:
 
 buildPythonApplication rec {
   pname = "protonvpn-gui";
-  version = "1.12.0";
+  # 4.1.0
+  version = "unstable-2023-10-25";
```

Comment:
I'm not that familiar with packaging python, but can you find out the required hashes of the dependencies by the commit hash of the main-application release?

If so and if Proton publishes the commit-hashes with version-numbers on release we should/could from an UX standpoint set the version to 4.1.0 because it would tell the user more in comparison to the official released version on the proton.me-page. WDYT?